caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java

```diff
@@ -220,7 +228,7 @@ public static void main(String[] args) {
             Runtime.getRuntime().addShutdownHook(new Thread(new ShutdownHook()));
 
             String[] dbInfo = dbParam.split("[.]");
-            Map<String, Object> daoConfig = new HashMap<String, Object>(2);
+            Map<String, Object> daoConfig = new HashMap<>(2);
             daoConfig.put("server", dbInfo[0]);
```

Comment:
How come the Map typing was removed on the HashMap?

Reply:
It's unnecessary as of Java 7.

Reply:
Okay, cool.